### ISSUE TITLE
*ngIf on a column causes additional call to UI Config Service

### ISSUE THREAD

**Issue by blackbaud-GavinNicol** [blackbaud/skyux-list-builder-view-grids#45]
### Expected behavior

When a list is configured for sticky settings you should be able to use an *ngIf with a column.

### Actual behavior

When a list is configured for sticky settings and a column is marked with an *ngIf then it is possible that when the *ngIf evaluates to true then an additional call is made to the UI Config Service resulting in the users sticky settings being overwritten.

### Steps to reproduce

1. Set up a list with sticky settings
2. Mark a column with an *ngIf that will evaluate to true (It's possible this step may need to take some time to evaluate in order to reproduce, perhaps an HTTP request)
3. Open the column chooser and un-check a couple columns.
4. Refresh the page
5. Notice the columns you unchecked are now visible and the network tab on the dev console shows an additional call to the UI Config service overwriting the sticky settings.

### Work around

Wrap the list in a div with an *ngIf that evaluates to true once your HTTP request is finished.

**Comment by Blackbaud-DanMiller** [comment]
Closing issue due to inactivity. If you believe this should still be addressed, please file a new issue and include any additional context and justification. Thank you.
